front/src/styles/scss/applications/stdcm/_results.scss

```
@@ -1,6 +1,6 @@
 .stdcm-results {
   background-color: rgb(233, 239, 242);
-  padding: 29px 30px 32px 32px;
+  padding: 29px 32px 32px 32px;
```

I think you can do this :)
```suggestion
  padding: 29px 32px 32px;
```

---

Done